gym/envs/robotics/robot_env.py

```diff
@@ -76,6 +76,7 @@ def reset(self):
         # Gimbel lock) or we may not achieve an initial condition (e.g. an object is within the hand).
         # In this case, we just keep randomizing until we eventually achieve a valid initial
         # configuration.
+        super(GoalEnv, self).reset()
```

Comment:
Is this the same as `super().reset()`?

Reply:
yes, but only for python 3, so let's keep it as is for python 2.7 compatibility